### DISCUSSION TITLE
New data 2024?

### DISCUSSION THREAD

**Discussion by Qbsol** [theOehrly/Fast-F1#532] [Q&A] [answered]
Hi can i expect new data from 2024 FP1 FP2 Bahrain ? How quick database is updated after event?

**Comment by Casper-Guo** [comment, marked as answer]
Assuming nothing has been unwittingly broken on our end or theirs during the offseason (testing seems to be fine), those data will definitely be available

I usually give it about 1.5 to 2 hours after a session ends before I try to get the data
